docs/rabbitmq.config.example

```diff
@@ -572,8 +667,12 @@
 
    %% TCP/Socket options (as per the broker configuration).
    %%
-   %% {tcp_listen_options, [{backlog,   128},
-   %%                       {nodelay,   true}]}
+   %% {tcp_listen_options, [
+   %%                         {backlog,   128},
+   %%                         {nodelay,       true},
```

Comment:
`nodelay` is forced by Ranch even if you disable it. I see no reason to do that, by the way.

Reply:
Should `nodelay` be removed from [PROJECT_ENV](https://github.com/rabbitmq/rabbitmq-server/blob/stable/Makefile#L60)?